geps/gep-2648/metadata.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: internal.gateway.networking.k8s.io/v1alpha1
+kind: GEPDetails
+number: 2648
+name: Direct Policy Attachment
+status: Experimental
+# Any authors who contribute to the GEP in any way should be listed here using
+# their Github handle.
+authors:
+  - robscott
+  - youngnick
```

Nit: I think you're the primary author of policy GEPs at this point
```suggestion
  - youngnick
  - robscott
```